integrations/cassandra/src/cassandra.go

```diff
@@ -17,6 +17,7 @@ type argumentList struct {
 	Username   string `default:"" help:"Username for accessing JMX."`
 	Password   string `default:"" help:"Password for the given user."`
 	ConfigPath string `default:"/etc/cassandra.yaml" help:"Cassandra configuration file."`
+	Timeout    int    `default:"1000" help:"Timeout in miliseconds per single JMX query."`
```

Comment:
Sorry, there is a typo here for `milliseconds`. Could you fix it, please?

Also, as we know we're having timeout issues using `1 second`, maybe could make sense to increase the default value. What do you think? Maybe 2 or 3 seconds could be ok. 

Thanks!